tests/commands/test_commit_command.py

```diff
@@ -109,6 +109,32 @@ def test_commit_command_with_dry_run_option(config, mocker: MockFixture):
         commit_cmd()
 
 
+@pytest.mark.usefixtures("staging_is_clean")
+def test_commit_command_with_write_message_to_file_option(
```

Comment:
let me know if i should also create a negative test cases where the option is getting passed no value or an invalid value